src/components/highlighter/types.ts

```diff
@@ -0,0 +1,51 @@
+import * as React from 'react';
+export type Token = {
+  types: string[];
+  content: string;
+  empty?: boolean;
+};
+
+export type StyleObj = {
+  [key: string]: string | number | null;
+};
+
+export type TokenOutputProps = {
```

Comment:
Can be interfaces. Even though this works it's a much more common pattern to save `type` for when creating new types of existing interfaces. The TS docs uses them this way.